image/base/install-pngcrush

```diff
@@ -2,7 +2,7 @@
 set -e
 PNGCRUSH_VERSION=1.7.92
 cd /tmp
-curl -O http://iweb.dl.sourceforge.net/project/pmt/pngcrush/$PNGCRUSH_VERSION/pngcrush-$PNGCRUSH_VERSION.tar.gz
+wget http://iweb.dl.sourceforge.net/project/pmt/pngcrush/$PNGCRUSH_VERSION/pngcrush-$PNGCRUSH_VERSION.tar.gz
```

Comment:
Getting redirected here. I could pass `-L` to curl but all the other scripts are using `wget` so I'm making it consistent